.gitignore

```diff
@@ -1,2 +1,4 @@
-/composer.lock
+/tests/*/*Tester.php
 /vendor/
+/tests/_output/*
+!/tests/_output/.gitkeep
```

Comment:
force adding also matches how the convention for the other repos, I don't think we use any `!` overrides (or at least try to avoid them)